nikola/plugins/command/import_wordpress.py

```diff
@@ -169,7 +169,11 @@ def show_info_about_mising_module(modulename):
 
         # Add tag redirects
         for tag in self.all_tags:
-            tag = utils.slugify(tag.decode('utf8'))
+            # In python 2, path is a str. slug requires a unicode
+            # object. According to wikipedia, unquoted strings will
+            # usually be UTF8
```

Comment:
This piece is essentially a copy'n'paste from somewhere else in the same source file. The original comment has no link to Wikipedia, either.